src/Pages/EditBlog.jsx

```diff
@@ -19,6 +19,7 @@ const EditBlog = () => {
 
   return (
     <Layout>
+      <h3>Edit blog</h3>
```

Comment:
Apu, may I know the reason of adding a new Header on this test PR?

Reply:
@fza-wtag bhai, I used this component to test protected route by checking if the text `Edit blog` is in the document.